parsing_scripts/wb_parser.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'account'
+require_relative 'transactions'
+require 'pry'
+require 'watir'
+require 'json'
+require 'json/pure'
+
+class WBParser
+  
+  @@wb_link = 'https://wb.micb.md/'
```

Comment:
class attributes are actually weird thing in ruby, I suggest you using CONSTANTS for such purposes, let me know when you face a case where constants won't suit you

Reply:
Done